ProcessLib/TES/TESReactionAdaptor.cpp

```diff
@@ -52,138 +52,138 @@ std::unique_ptr<TESFEMReactionAdaptor> TESFEMReactionAdaptor::newInstance(
 
 TESFEMReactionAdaptorAdsorption::TESFEMReactionAdaptorAdsorption(
     TESLocalAssemblerData const& data)
-    // caution fragile: this relies in this constructor b eing called __after__
```

Comment:
Comment's, not containing private member references also should not be changed.

Reply:
Recovered.